modules/dxp/apps/commerce-avalara/commerce-avalara-tax-engine-fixed-web/src/main/resources/content/Language.properties

```diff
@@ -0,0 +1,8 @@
+account-number=Account Number
+avalara=Avalara
+avalara-tax-rate-description=Avalara Type Rate Description
```

Comment:
@CalvinKeum should this be `Avalara Tax Rate Description`?

Reply:
Looking at the other examples, I suppose it can be renamed to "Avalara Tax Rate". I can go either way.

Reply:
Oh just noticed its a *-description Lang key. So it doesn't necessarily need to have a matching key and value. Should be fine as is then.

Reply:
oh noticed this is a *-description lang key. So the key and value does not have to necessarily match. yeah should be fine then ahha.